src/main/java/uk/gov/justice/hmpps/prison/service/transfer/PrisonTransferService.kt

```diff
@@ -125,12 +155,23 @@ class PrisonTransferService(
   }
 
   private fun getCellLocation(cellLocation: String?, prison: AgencyLocation): Result<AgencyInternalLocation> {
-    val internalLocationCode = cellLocation ?: "${prison.id}-RECP"
+    return getCellLocation(cellLocation, prison.id)
+  }
+
+  private fun getCellLocation(cellLocation: String?, prisonCode: String): Result<AgencyInternalLocation> {
+    val internalLocationCode = cellLocation ?: "$prisonCode-RECP"
     return agencyInternalLocationRepository.findOneByDescriptionAndAgencyId(
```

Comment:
in the future we can look at whether there is a better way of finding a prison's reception location